docs/shortfin/sd/sdxl_rdna_demo.md

```diff
@@ -0,0 +1,200 @@
+
+```shell
+mkdir demo
+cd demo
+```
+
+### Install ROCm Community Edition
+
+Simply download the matching tarball, extract it, and add it to your environment variables:
+
+```shell
+mkdir rocm
+cd rocm
+# Check GitHub releases for other distributions.
+wget https://therock-artifacts.s3.us-east-2.amazonaws.com/therock-dist-gfx1201-20250305.tar.gz
+tar -xzf therock-dist-gfx1201-20250305.tar.gz
+export PATH="$PWD/bin:$PATH"
+export LD_LIBRARY_PATH="$PWD/lib:$LD_LIBRARY_PATH"
+cd ..
+```
+
+You can find other ROCm Community releases in the [TheRock repository](https://github.com/ROCm/TheRock).
+
+Confirm that your GPU is detected:
+```console
+~/demo
+➜  which rocm-smi
+/home/jakub/demo/rocm/bin/rocm-smi
+~/demo
+➜  rocm-smi
+======================================== ROCm System Management Interface ========================================
+================================================== Concise Info ==================================================
+Device  Node  IDs              Temp    Power  Partitions          SCLK  MCLK   Fan    Perf  PwrCap  VRAM%  GPU%
+              (DID,     GUID)  (Edge)  (Avg)  (Mem, Compute, ID)
+==================================================================================================================
+0       2     0x7550,   37870  32.0°C  3.0W   N/A, N/A, 0         0Mhz  96Mhz  0%     auto  0.0W    5%     0%
+1       1     0x7448,   7019   31.0°C  7.0W   N/A, N/A, 0         0Mhz  96Mhz  20.0%  auto  241.0W  0%     0%
+==================================================================================================================
+============================================== End of ROCm SMI Log ===============================================
+```
+
+Above, `rocm-smi` lists two GPUs: a Radeon RX 9070 and a Radeon Pro W7900.
+
+### Install SHARK AI and Shortfin
+
+First, create and activate a Python Virtual Environment:
+
+```shell
+python -m venv venv
+source venv/bin/activate
+```
+
+Clone the shark-ai repository and install Shortfin and its dependencies:
+
+```shell
+git clone https://github.com/nod-ai/shark-ai
+cd shark-ai
+git switch shared/rdna4
+```
+
+Install pip requirements:
+```shell
+pip install -r requirements-iree-pinned.txt
+pip install -r pytorch-cpu-requirements.txt
+pip install -r requirements.txt
+
+cd shortfin
+pip install -e .
+```
+
+## Start Shortfin and run SDXL
+
+Start the Shortfin server with the correct target (`gfx1100` for RDNA3, `gfx1201` for RDNA4).
+You can override the network port used using the `--port <PORT-NUM>` flag.
+
+### FP8: RDNA4 only
+
+Note that the first run will download all the artifacts necessary (the model code and the weights).
+This may take a while. The subsequent runs will use the artifacts cached in `~/.cache/shark/genfiles/sdxl`.
+
+```shell
+python -m python.shortfin_apps.sd.server --device=amdgpu --target=gfx1201 --build_preference=precompiled \
+  --device=hip --device_ids 0 --model_config=sdxl_config_fp8_ocp.json
```

Comment:
Does this file exist? There is a `sdxl_config_fp8.json` at `shortfin/` but I would expect all SDXL files to be stored at `shortfin/python/shortfin_apps/sd/` (or this `docs/shortfin/sd/` folder)

Reply:
It doesn't need to exist -- it will be pulled from azure if it doesn't exist.

Reply:
Cool. Wasn't sure if the `_ocp` part of the file path was leftover from some previous naming, given that the file in the repo doesn't have it.